docker-compose.yml

```diff
@@ -22,15 +25,16 @@ services:
     depends_on:
       - mongo
     environment:
-      MONGO_URL: "mongodb://mongo:27017/reaction"
-      MONGO_OPLOG_URL: "mongodb://mongo:27017/local"
+      MONGO_URL: "mongodb://mongo.streams.reaction.localhost:27017/reaction"
+      MONGO_OPLOG_URL: "mongodb://mongo.streams.reaction.localhost:27017/local"
       ROOT_URL: "http://localhost:3000"
       HYDRA_ADMIN_URL: "http://hydra:4445"
       HYDRA_OAUTH2_INTROSPECT_URL: "http://hydra:4445/oauth2/introspect"
     networks:
       default:
       api:
       auth:
+      streams:
```

Comment:
If the default network is used for the Mongo connection then this service doesn't need to be on the streams network.